config/settings/base.py

```diff
@@ -369,6 +373,18 @@
 }
 
 
+# OIDC Authentication
+# https://mozilla-django-oidc.readthedocs.io/
+OIDC_AUTH_ENABLED = strtobool(os.getenv("CORGI_OIDC_AUTH_ENABLED", "false"))
+OIDC_OP_JWKS_ENDPOINT = os.getenv("CORGI_OIDC_OP_JWKS_ENDPOINT", "")
```

Comment:
Could you perhaps open a component-registry-ops PR with this values populated so that we could know how to test this code in DEV? It will be required for this feature to work in stage anyway.

I found only one value for client secret in the Vault.

Reply:
Because SSO restricts the URIs that can receive tokens, you can't use the stage values for local testing. I've added the configuration for stage and prod to component-registry-ops.